### ISSUE TITLE
Buildcraft Quarries Will Stop Creating the Frame Mid-Construction

### ISSUE THREAD

**Issue by Miradden** [xJon/The-1.12.2-Pack#265]
**Basic details**
1. Are you using a legitimate launcher and Minecraft account:
yes
2. Are you using a computer with 8GB or more of RAM? ([How to check](https://www.wikihow.com/Check-Computer-RAM)):
 yes
3. Are you using Java 8, 64-bit? ([How to check](https://www.wikihow.com/Determine-Java-Version)):
yes

**Information required**
1. Crash/latest log *link* ([Where to find](https://github.com/xJon/The-1.12.2-Pack/blob/master/log_location.md#finding-modpack-installation-folder)): https://paste.ubuntu.com/p/Q3Q3JPsKzb/
2. Is Optifine installed or any other additional mods: no 

**Issue description**
Tried to setup Buildcraft Quarry in Aroma1997's mining dimension.
setup was:

Thermal Expansion Magmatic Dynamos > redstone fluxduct > FE Consumer > Energy Bridge > MJ Producer > wooden kinesis pipe > golden kinesis pipe > Quarry (64x64) + ender chest feeding to an ME.

the second I placed the quarry the game crashed and the mods it said were responsible were all the assorted buildcraft related mods (Buildcraft, Buildcraft Silicon, Buildcraft Compat, etc.) as well as Galacticraft. It booted me to the title screen, so I re-connected to the server and the quarry had been placed and it started mining, business as usual. After it had mined ~5 layers I decided to add a resonant energy cell to buffer power, and when I disconnected the power from the quarry I was never able to get it to start back up again. tried from the dynamos, from just the cell, both, new pipes, new converters, etc and nothing changed. tried different dimensions of quarry and nothing. best it does is when i place a brand new quarry to power it will construct ~90% of the frame then quit and the blue energy in the BC pipes will flash twice before going dark. This happens in both mining dimension and overworld. Quarries work fine with the same setup in creative singleplayer. Have restarted both client and server. Quarries worked fine before with the same setup, although they had always been in the overworld. Hope I can find a way to fix this.

**Comment by xJon** [comment]
Recently I've forwarded some quarry related issues to the mod devs and apparently they fixed some, but the update was not released yet. I will forward this crash report as well.
Can you for now just place it again, maybe somewhere else? Or is it unusable for you?

**Comment by Miradden** [comment]
Quarries do not currently work in any form on the server.

**Comment by xJon** [comment]
Which server?

**Comment by Miradden** [comment]
Private server I am running for myself and my friends

**Comment by xJon** [comment]
Do they work for you in other servers or single player worlds?

**Comment by Miradden** [comment]
yeah works fine in singleplayer creative with the exact same setup

**Comment by JBlackJr** [comment]
@xJon @Miradden, I figured it out. So I have almost the exact same set up as you, Miradden. I was converting RF to MJ using Thermal Expansion Compression Dynamos. I has 6 of them at first and it was working fine. As I began upgrading the Dynamos and adding more of them, my Quarry would stop building the frame if I marked out a large area. If the default size was chosen, it would complete the frame but stop digging very shortly. Some MJ would get to the quarry, and store it in the buffer, but then it wouldn't accept any more power until broke and placed back down.

So I tried using just BuildCraft machines to power it. Using the creative engines, I set one engine next to the quarry at 256 mj tick rate and the quarry works fine. But when I add another engine and set the tick rate to 256, the quarry does the same thing as before. 

So, in my set up, I have 12 Compression engines operating at 500 RF/tick. That's 6000 RF/tick total. The 10:1 conversion rate puts that to 600 MJ/tick. Guessing there is an upper limit where the quarry can operate safely around 384 MJ/tick.  My setup breaches that upper limit and the quarry bugs out. 

My fix was to put down a thermal expansion energy cell and set the output tick rate to 3000 RF/tick and I haven't been able to replicate the issue with those settings. Hope this helps.

**Comment by xJon** [comment]
Should be fixed in `1.1.2`.
